### DISCUSSION TITLE
The ongoing disaster of the standard model of particle physics

### DISCUSSION THREAD

**Discussion by sahil5d** [a-cosmology-group/acg#201] [General]
This video from Fermilab on the standard model of particle physics is a disaster.

https://www.youtube.com/watch?v=Wg24PovZTXM&ab_channel=Fermilab

Here are a few lines that are very disconcerting, in that Fermilab puts out such content, and that the SM Lagrangian is the epitome of particle physics.

- "But it's kind of fun to see what a mess the equation can be."
- "It describes what are called ghost particles, which aren’t true particles, but they are a mathematical construct that fixes some unwanted predictions of the theory."
- "And this fifth section is more of that ghost cleanup... Again, these are introduced to maintain the integrity of the calculation."
- "If you want to dig more into these ghost particles, make sure to buy some aspirin first. They always gave me headaches when I had to calculate them."
- "If you want to dig more deeply into it, well, you have a fair bit of studying in front of you. I mean I suppose I could do it for you, but I think this would be a great time to duck that responsibility."
- "I know what I’ll do. I’ll do what my professors used to tell me when I asked these kind of questions and say 'the solution is left as an exercise for the student.'"

_

I am curious as to what ACG thinks the future is for particle physics. In what direction should this branch of science go?

I think the future is Weber's formula (Coulomb's 1/r^2 function with a velocity-dependent term and an acceleration-dependent term). André Assis has previously explained how it might enable understanding of the nucleus in how like charges attract at very very small distance.

**Comment by HanDeBruijn** [comment]
Related stuff is found in [thread 200](https://github.com/orgs/a-cosmology-group/discussions/200).
Quite in general, I do not understand how a **standard** understanding of nature can be conceived.
It's the same idiocy as if it would be possible to decide [by legislation](https://en.wikipedia.org/wiki/Indiana_pi_bill) that the standard value of $\pi$ is $3.0$.
>I think the future is Weber's formula

I don't think so but anyway it is formula (2.6) on page 17 in the book _Origin of Inertia_ by Amitabha Ghosh:

$$
F = \frac{e e'}{4\pi\epsilon r^2}\left[1-\frac{1}{c^2}\left(\frac{dr}{dt}\right)^2+\frac{2r}{c^2}\frac{d^2r}{dt^2}\right]
$$

**Comment by RedshiftDrift** [comment]
> I am curious as to what ACG thinks the future is for particle physics. In what direction should this branch of science go?

Same direction as cosmology, in the garbage can.

**Reply by MJoseph315** [reply]
Louis, what specifically are your problems with the standard model of particle physics?

**Reply by HanDeBruijn** [reply]
>Louis, what specifically are your problems with the standard model of particle physics?

I don't want to interfere, but this sounds like the common 'who is to be blamed' management trick.
Employee: "The **system** doen't work properly". Employer: "Ah, **you** have problems with **our** system".

**Reply by RedshiftDrift** [reply]
Sabine Hossenfelder has made several videos explaining "the" problems (not my problems) with particle physics.  The talk she gave at the Universität Stuttgart explains the problems very well (in German with English subtitles) [Was läuft falsch in der gegenwärtigen Physik?](https://www.youtube.com/watch?v=99hVAu1k6G8)  Her YouTube channel has many more videos explaining why the standard model of particle physics doesn't work.

Essentially, we have the following progression of knowledge:
1. chemistry which studies small entities that have been detected: molecules,
2. atomic physics which studies the components of interacting molecules: atoms, and photons, all of which have also been detected,
3. nuclear and plasma physics which studies the parts of atoms such as electrons and nuclei and their components, protons and neutrons and other fermions and bosons, all of these particles have been detected.

But then...
4. particle physics which studies the mathematical symmetries of protons, neutrons, and other fermions and bosons, symetries such as U(1), SU(2), etc. and various combinations like U(1) $\times$ SU(2) $\times$ SU(3) that make up what's called quarks and strings in 10-dimensional space...

 ***None*** of those have been detected and theorists have made up reasons explaining why they cannot be seen!

**Reply by budrap00** [reply]
Louis,

I generally agree with you here but for the characterization of bosons as having been detected. Of the 5 elementary bosons, only the photon can be said to be detected but the photon is not a particle; it is a wave quanta - the minimum energy of one full wavelength for a given frequency. The other 4 elementary bosons (W, Z, Gluon, Higgs) have never been detected although a model-dependent claim to having detected their "decay products" is passed off as empirical evidence for their existence.

**Reply by RedshiftDrift** [reply]
That's why I put the photon with 2. atomic physics.

**Reply by budrap00** [reply]
The elementary bosons being in 3 instead of 4 is what I was getting at - they are not directly detected, only inferred.

**Comment by LifeIsStrange** [comment]
Quantum physics is a non-causal model filled with many strangeness and paradoxes but as always the map isn't the territory.
There exists three paradigms that attempts to bring causality and or to unify the microscopic with the classical macroscopic theories.
1) The classical unified fields theories
2) the semiclassical theories (hybrids)
3) The hidden variables theories

Among them, 1) are not failures contrary to the popular myth, there are formalisms that do unify electromagnetism and gravity under a single representation. Some theories such as Schrodinger have highly intriguing properties, as in his formalism, the cosmological constant "naturally emerge". And the cosmological constant is the most well known and simple way to modelize antigravity/dark energy which is a necessary component of most stably static universes.
However they are in a state of abandon despite some of their early results.

But semiclassical theories are an immense empirical success, that demystifies most quantum phenomenas, are macroscopically valid and shows that the assumed so called classical limit is propagandist fraud.
The most popular semiclassical theory is stochastic electrodynamics (which ground for the first time inertia which cannot be quantized like gravity, and also uniquely rederive the second law of newton), but there are many other insightful representations as is well outlined in this state of the art review:
https://www.researchgate.net/publication/374257062_The_zitterbewegung_electron_puzzle
see also https://www.researchgate.net/publication/13379419_Inertia_as_a_zero-point-field_Lorentz_force

One may ask how does this matters to cosmology?
Well it can in many ways, one being the cosmological constant problem ("worst prediction in history"), two, while many try to explain the rotation curves of galaxies as missing matter or as a modified gravity (e.g. MOND), it is almost always forgotten that there is a third possibility, that of modified inertia.
One of the earliests advocates of modified inertia that I know of is Milgrom (2005) (albeit I wonder if the topic was explored during the 20th century)
https://arxiv.org/abs/2310.14334
The only modified theory of inertia that I know of that has been empirically tested is Quantized Inertia
https://www.researchgate.net/publication/380815611_Testing_Quantised_Inertia_on_Proxima_Centauri
https://www.researchgate.net/publication/334708215_Testing_NewtonGR_MoND_and_quantised_inertia_on_wide_binaries
Sadly the theory has not yet been tested on GAIA DR3.

Quantized inertia is interesting because:
It explore modified inertia, a unique mechanism, the third paradigm.
It has passed all its current observational tests (while MOND has not) albeit arguably the theory is undertested until someones do GAIA DR3
It has apparently zero free parameter (while MOND has one)
Uniquely it has the ambition to derive its law from quantum theory while both dark matter and MOND have no credible mechanism besides magic/we don't know (well there are axions, etc but those are weak)
And it join cosmology with the microscopic world, ingeniously via the highly mysterious casimir effect
https://en.wikipedia.org/wiki/Casimir_effect
( both the casimir effect and inertia are shown to derive from the zero point field)
the unruh radiation (that has apparently been observed recently)
and the rindler horizons
While the exact mathematical theory might be wrong (and indeed a review paper shows that some corrections are needed) linking quantum theory and the casimir effect/rindler horizons to theories of modified inertia is an intriguing idea and its empirical validity in cosmology is something to keep an eye at.
Moreover such a theory of inertia might also be linked to insights via the zero point field and stochastic electrodynamics, also there are key empirical deviations from general relativity that could give major insights.
Being, the Allais effect, 
https://www.researchgate.net/publication/351853011_The_Allais_Effect_from_Quantised_Inertia
the sagnac effect square variant (that nobody has attempted despite how simple it is)
https://www.scirp.org/journal/paperinformation?paperid=91858
and the most important and proven deviation from general relativity:

The mossbauer rotor experiment that has been reproduced many times and consistently shows an excess redshift of 20% 
Since the mossbauer effect is liked to the effect of acceleration and quantized inertia of minimal acceleration, I wonder if the mossbauer effect could be explained via modified inertia, an Idea that nobody has yet considered
https://epjplus.epj.org/articles/epjplus/abs/2015/10/13360_2015_Article_922/13360_2015_Article_922.html
While Corda relativist explanation has been highly criticized, he has made a paper that might interest you @RedshiftDrift 
In my neophyte opinion, the excess redshift observed in the mossbauer effect is valid at cosmic scales and is IMO a new empirically proven in lab, tired light mechanism (20% of the effect at tested speed and size), I have no idea what its magnitude/relevance would be but I believe its effect in cosmology is not negligible, especially since it affect rotational systems (and hence stars and galaxies, maybe even clusters)
While I disagree with Corda explanation he does makes this parallel in this paper
https://arxiv.org/abs/2203.02282
It is IMO a key new underlooked topic in cosmology, that I hope you will look deeper into than I did. If I understand correctly it is a mechanism of excess radiation from acceleration.

**Reply by RedshiftDrift** [reply]
Thanks @LifeIsStrange for the references to papers on quantum physics.  In my previous life (before 2006) this topic was of great interest to me.  I was never impressed by the "Zitterbewegung" model...  As for the "zero-point energy" resulting from quantization, this is for me a mathematical artifact without any physical meaning.  (*The expression for the quantized energy* $(a^\dagger a + a a^\dagger)/2$ *is written as* $a^\dagger a + \frac{1}{2}$, *but this does not work since the lowering operator applied on the empty state* $a|0\rangle$ *makes no physical sense.  As a result*  $a a^\dagger|0\rangle = |0\rangle$, *but the energy* $a^\dagger a|0\rangle$ *has no physical meaning.  Therefore, the zero-point energy term* $... + \frac{1}{2}$ *that appears from the commutator cannot have a physical meaning either.  A "cosmological constant" too big by a factor* $10^{120}$ *("worst prediction in history") is irrelevant, it's not a physical prediction.*)  I was not aware of the Mössbauer rotor discrepancy, but it seems that the problem has been resolved 'theoretically'.  Our better understanding of a physical effect doesn't change the actual physical effect, just like understanding that the Earth is going around the sun did not suddenly push the Earth to set it in an orbit around the sun.  In the same way, a correct understanding of the Mössbauer rotor does not make the "cosmological constant" a physical reality.

As an astrophysicist (since 2006), I believe these mathematical details of quantum theory cannot be translated into a solution for cosmology. (Same thing for relativity, a tweak on relativity theory won't give us the cosmological solution we are looking for.)

You must have read, on this discussion forum, various incomplete descriptions of my tired-light model based on established QED theory.  There is no space expansion, the Stimulated Transfer Redshift (STz) model produces a $ln(1+z)$ redshift-distance relationship.  The so-called "cosmological constant" is reproduced by STz by fitting the two parameters $\Omega_M$ and $\Omega_\Lambda$ of the $\Lambda \mbox{CDM}$ function to the $ln(1+z)$ function.  This gives a "cosmological constant" $\Omega_\Lambda = 0.69$ in agreement with accepted values (see graph below) as a result of the mathematical properties of the two functions.  The "cosmological constant" is not a physical phenomenon.

@LifeIsStrange for a neophyte you seem to have an extensive knowledge of quantum physics and gravity theories.  Would you mind sharing a bit more info about yourself?

![LCDM-fit-to-exp](https://github.com/a-cosmology-group/acg/assets/16748402/35ac37b9-aa8e-45a5-a086-167c4cc1d1be)
